java/msbuild/ice.proj

```diff
@@ -26,14 +26,14 @@
   </Target>
 
   <Target Name="BuildDist" DependsOnTargets="slice2java">
-    <Exec Command="$(Gradle) $(GRADLEARGS) dist -PcppPlatform=$(Platform) -PcppConfiguration=$(Configuration)"
+    <Exec Command="$(Gradle) $(GRADLEARGS) dist -x rewriteDryRun -x checkstyle -PcppPlatform=$(Platform) -PcppConfiguration=$(Configuration)"
```

Comment:
I know I recommend to put these on dist but now I'm actually not sure that it's required.

Reply:
We should remove `-x checkstyle`